analysis/org.eclipse.tracecompass.analysis.counters.core/src/org/eclipse/tracecompass/analysis/counters/core/aspects/AbstractCounterAspect.java

```diff
@@ -130,6 +130,7 @@ public boolean equals(@Nullable Object obj) {
      * @return the type of this counter
      * @since 2.1
      */
```

Comment:
Now that it overrides, you can remove the Javadoc here, the one from interface will be used.

Reply:
not fixed

Reply:
It's the Javadoc of getType() that can be removed.